2017-bae-europe/french_transition/french_transition.ipynb

```diff
@@ -142,17 +146,501 @@
   },
   {
    "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get where_fuel\n",
+    "# Wait for the complete message to move on\n",
+    "filename = 'astrid.sqlite'\n",
+    "\n",
+    "cur = an.get_cursor(filename)\n",
+    "init_year, init_month, duration, timestep = an.get_timesteps(cur)\n",
+    "agent_ids = an.get_prototype_id(cur, 'uox_mixer')\n",
+    "fuel_from_uox = an.facility_commodity_flux(cur, agent_ids, ['mox'], True, True)\n",
+    "agent_ids = an.get_prototype_id(cur, 'mox_mixer')\n",
+    "fuel_from_mox = an.facility_commodity_flux(cur, agent_ids, ['mox'], True, True)\n",
+    "dict = {}\n",
+    "dict['fuel from spent uox'] = fuel_from_uox['mox']\n",
+    "dict['fuel from spent mox'] = fuel_from_mox['mox']\n",
+    "an.stacked_bar_chart(dict, timestep,\n",
```

Comment:
Again, many plots are produced here, but I only see a few of them in the bottom.